src/sentry/integrations/slack/tasks/post_message.py

```diff
@@ -0,0 +1,55 @@
+from __future__ import annotations
+
+import logging
+from collections.abc import Mapping
+from typing import Any
+
+from sentry.integrations.slack.service import SlackService
+from sentry.silo.base import SiloMode
+from sentry.tasks.base import instrumented_task
+
+logger = logging.getLogger("sentry.integrations.slack.tasks")
+
+
+# TODO: add retry logic
+@instrumented_task(
+    name="sentry.integrations.slack.tasks.post_message",
+    queue="integrations",
+    max_retries=0,
+    silo_mode=SiloMode.REGION,
+)
+def post_message(
+    integration_id: int,
+    payload: Mapping[str, Any],
+    log_error_message: str,
+    log_params: Mapping[str, Any],
+) -> None:
+    service = SlackService.default()
+    service.send_message_to_slack_channel(
+        integration_id=integration_id,
+        payload=payload,
+        log_error_message=log_error_message,
+        log_params=log_params,
+    )
+
+
+# TODO: add retry logic
+@instrumented_task(
+    name="sentry.integrations.slack.post_message_control",
```

Comment:
This task and `sentry.integrations.slack.tasks.post_message` both have duplicate tasks registered. We could remove the task defined in `src/sentry/tasks/integrations/slack/post_message` and only do an import shim to avoid the duplicate task names.

Reply:
what does duplicate task registered mean? Is that like two separate tasks are named the same ? 

Also for the import shim, would that look something like the class/symbol import shims ? I'm kinda confused rn, don't the task import shims need the `@instrumented_task` decorator to be valid?

Reply:
> what does duplicate task registered mean? Is that like two separate tasks are named the same ?

Yes, two tasks have the same `name` parameter provided to `instrumented_task`. Celery will get confused when trying to execute these tasks and we should avoid that.